scanomatic/data_processing/pheno/state.py

```diff
@@ -50,10 +50,15 @@ def serialized(self) -> list[Union[str, int, float]]:
             self.linear_regression_size,
             None if self.phenotypes_inclusion is None
             else self.phenotypes_inclusion.name,
-            self.no_growth_monotonicity_threshold,
-            self.no_growth_pop_doublings_threshold
+            None if self.no_growth_monotonicity_threshold is None
+            else self.no_growth_monotonicity_threshold,
+            None if self.no_growth_pop_doublings_threshold is None
+            else self.no_growth_pop_doublings_threshold,
```

Comment:
what does this do? Also is serialized really used now, can't it be removed.

Reply:
Yes, it can safely be removed.

Reply:
fixed